### PR TITLE
Fixtest

### PR DESCRIPTION
**Pull request by Vectormike**
Fixed typo
Added transaction options
Check balance currency

### CODE REVIEW

**Comment by solomon-os** [comment]
Did you run the tests after the modifications?

**Comment by Vectormike** [comment]
Not yet. I'm still waiting for the keys.

**Comment by Vectormike** [comment]
Did you get the feedback from the team?

File: types.d.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ declare interface TransactionOptions {
     // A valid date format in string
     dateFrom?: string | Date;
     // A valid date format in string
-    dateTo?: '2020-01-15';
+    dateTo?: string | Date;
```
Comment (solomon-os) [inline types.d.ts:9]: Nice one man.
I didn't see this.
Thanks for this

**Review by solomon-os — COMMENTED** [review]
No.
You know what I'll give you my keys on whatsapp

**Comment by Vectormike** [comment]
No problem

**Pull request merged**